platform/darwin/src/MGLTileID.h

```diff
@@ -0,0 +1,19 @@
+#import <Foundation/Foundation.h>
+
+#import "MGLFoundation.h"
+
+typedef struct
+{
+  uint8_t z;
```

Comment:
is z actually zoom? if so, use "zoom" as the var name?

Reply:
This is following a convention set in the c++ equivalent of this class `CanonicalTileID`.